packages/lib-utils/package.json

```diff
@@ -36,6 +36,8 @@
     "redux-thunk": "^2.4.1"
   },
   "dependencies": {
+    "@redhat-cloud-services/frontend-components": "^3.9.10",
+    "classnames": "^2.3.1",
```

Comment:
Do we really use this in our code? If not, we can remove this explicit dependency.